data/sampleResume2.json

```diff
@@ -57,30 +57,6 @@
         "Redesigned our SaaS-based airport map UI following the industry standard to support up to 5x more data on the viewport.",
         "Analyzed post-release statistics and highlighted key takeaways to global senior management for continuous improvement."
       ]
-    },
```

Comment:
@stevahnes  forgot to update the format, I just remove the invalid format for sample data, feel free to update it